docs/user-tutorial/data-diagnosis.md

```diff
@@ -165,7 +179,15 @@ The function used for this rule.
 
 - `multi_rules`: the rule is to check if the combined results of multiple previous rules and metrics violate the criteria.
 
-  For example, if the 'criteria' is 'lambda label:True if label["rule4"]+label["rule5"]>=2 else False', the rule is that if the sum of labeled metrics in rule4 and rule5 is larger than 2, it should be defective.
+   - If the 'criteria' is 'lambda label:True if label["rule4"]+label["rule5"]>=2 else False', the rule is that if the sum of labeled metrics in rule4 and rule5 is larger than 2, it should be defective.
```

Comment:
-  'criteria' is 'lambda label:True if label["rule4"]+label["rule5"]>=2 else False', which means that this rule will be triggered if the sum of labeled metrics in rule4 and rule5 is larger than 2